src/services/__tests__/anchor-service.test.ts

```diff
@@ -69,25 +69,39 @@ function createStream(id: StreamID, log: CID[], anchorStatus: AnchorStatus = Anc
   }
 }
 
+const mockIpfsClient = new MockIpfsClient()
+jest.mock('ipfs-http-client', () => {
```

Comment:
@stephhuynh18 If this works, it means, the test runs in CJS mode, IMO, which is not right long term. FYI.

Reply:
instead of mocking this library to override the `create` function, we could make `IpfsService.init()` take an optional param of the IpfsClient to use, and just manually provide the MockIpfsClient in the unit test

Reply:
@ukstv I forgot to change this to `unstable_mockModule`. Does using this make the tests run in ESM mode?

Reply:
> Does using this make the tests run in ESM mode?

Based on how the tests are run, it seems they run in ESM mode already. I was just confused that the tests worked. In my experience, jest `mock` and `requireActual` do not work in ESM mode. If they do now, it is perfect.

Reply:
@stbrody will do that if the jest mocking messes things up later =) hopefully its actually fixed